sos/report/plugins/kubernetes.py

```diff
@@ -247,6 +247,24 @@ def postproc(self):
         # output that is not hit by the previous iteration.
         self.do_cmd_private_sub(self.kube_cmd)
 
+        self.do_file_private_sub('/*')
+
+        # clear the certificate data from files that does not have
+        # the certificate banners that the _private_sub() methods look for
+        _fields = [
+            '.*.crt',
+            'client-certificate-data',
+            'client-key-data',
+            'certificate-authority-data',
+            '.*.key',
+            'token',
+            '.*token.*.value'  # don't blind match `.*token.*` and lose names
+        ]
+
+        regex = r'(\s*(%s):)(.*)' % '|'.join(_fields)
+
+        self.do_path_regex_sub('/*', regex, r'\1 *******')
```

Comment:
Post processing needs to be done very intentionally on the files the obfuscation is built around.

Reply:
now limited to `config_files`, scrubbing  kubeconfig style